libraries/AP_HAL_ChibiOS/hwdef/AnyleafH7/defaults.parm

```diff
@@ -0,0 +1,25 @@
+# Onboard ELRS
+SERIAL5_PROTOCOL 23 # RCIN
+# RC_OPTIONS 512  # Suppress ELRS errors
+RC5_OPTION 154 # Set up arm switch to `ArmDisarm with AirMode`
```

Comment:
I would not advise this.....switches are very personal

Reply:
Normally, I'd agree, but ELRS hard-sets arm to channel 5, and not using airmode is IMO dangerous in any flight mode.

Reply:
I agree with Henry - there are other ways to do it. I have RC5_OPTION = 0. See my video: https://youtu.be/YO2yA1fmZBs

Reply:
and AirMode is potentially dangerous and unexpected, not having it active on arm is not...and many would want a different kind of arm switch...for example ARM/Emergency stop....I will note in the README that I am modifying now and will push shortly, about ELRS arming, but please remove this